documentation/website/src/handbook/integration/serenity-2-migration-guide.md

```diff
@@ -0,0 +1,688 @@
+
+So replace your previous crew import sections with the following:
+
+```javascript
+const {ConsoleReporter} = require('@serenity-js/console-reporter'),
+  {ArtifactArchiver} = require("@serenity-js/core"),
+  {Photographer, TakePhotosOfInteractions} = require('@serenity-js/protractor'),
+  {SerenityBDDReporter} = require("@serenity-js/serenity-bdd");
+```
+
+Also remove whatever was in the `crew []` section in the `serenity` object as this all needs to be replaced.
+
+Make the following changes in your config object so that it looks like this:
+
+```javascript
+exports.config = {
+    chromeDriver: require('chromedriver/lib/chromedriver').path,
+    SELENIUM_PROMISE_MANAGER: false,
+    directConnect: true,
+    allScriptsTimeout: 11000,
+    framework:      'custom',
+    frameworkPath:  require.resolve('@serenity-js/protractor/adapter'),
+    specs: [ 'spec/*.spec.ts', ],
+    serenity: {
+        crew: [
+          ArtifactArchiver.storingArtifactsAt("./target/site/serenity"),
+          Photographer.whoWill(TakePhotosOfInteractions),
+          new SerenityBDDReporter(),
+          new ConsoleReporter.withDefaultColourSupport(),
+        ],
+    },
+    // other Protractor config (see Runner specific configurations below)
+    };
+ ```
+
+### The new ConsoleReporter
+The new ConsoleReporter supports several colour themes, which you can choose by calling either of the following factory methods:
+```javascript
+ConsoleReporter.withDefaultColourSupport()
+ConsoleReporter.forMonochromaticTerminals()
+ConsoleReporter.forDarkTerminals()
+ConsoleReporter.forLightTerminals()
+```
+
+### Runner specific configurations
+#### Mocha
+Note that Mocha is not useable yet with 2.x, so if you previously used mocha you'll need to switch to an alterntive runner for the time being. 
+Jasmine is recommended as the switch to it should be trivial. There are very few differences between how mocha and jasmine specs now look, 
+given that assertions for all runners will use the new `@serenity-js/assertions`, exposing a uniform `Ensure` API.
+So if you previously used mocha then remove your `mochaOpts` section:
+
+```javascript
+ mochaOpts: {
+    ui: "bdd",
+    compiler: "ts:ts-node/register",
+  },
+```
+
+#### Cucumber
+If you use Cucumber, you'll need a section similar to this: 
+```javascript
+ cucumberOpts: {
+        require: [
+            'features/step_definitions/**/*.ts',
+            'features/support/*.ts',
+        ],
+        'require-module': ['ts-node/register'],
+        tags: [],
+    },
+```
+Make sure you add a folder (e.g. support) and reference a setup.ts file this within the require array section where you can add any global configuration and setup (explained below)
+#### Jasmine
+If you use Jasmine, you'll need a section similar to this:
+```javascript
+ jasmineNodeOpts: {
+    requires: [
+      "ts-node/register",
+      path.resolve(__dirname, "node_modules/@serenity-js/jasmine"),
```

Comment:
If we're using Protractor and have registered the the Protractor adapter via
```
frameworkPath:  require.resolve('@serenity-js/protractor/adapter'),
```
we don't need the `path.resolve(__dirname, "node_modules/@serenity-js/jasmine")`.

We did in `alpha`, but 2.0 doesn't require you to jump through this hoop :)